src/utils/extenders.js

```diff
@@ -0,0 +1,24 @@
+const { Message } = require("discord.js")
+const responses = require("./responses")
+
+Message.prototype.error = function (string, options = {}) { // allows error messages to be sent with a standard response and emoji
+    options.prefixEmoji = "error";
+    string = responses.error[string] ? responses.error[string] : string;
+    return this.sendM(string, options);
+};
+
+Message.prototype.success = function (string, options = {}) { // allows success messages to be sent with a standard response and emoji
+    options.prefixEmoji = "success";
+    return this.sendM(string, options);
+};
+
+Message.prototype.sendM = function (string, options = {}) { // sends a message with options/edits the message
```

Comment:
I would make this `sendMessage`. Makes it more clear, dont sacrifice readability for a shorter name.

Reply:
Changed.